### PR TITLE
Production

### PR DESCRIPTION
**Pull request by justinkonecny**
Lots of changes for production deploy
 - Create `scripts/` directory and move `build.sh`
 - Update `Dockerfile`
 - Add `Dockerrun.aws.json`
 - Change server port from `8090` to `80`
 - Add `deploy.sh`

### CODE REVIEW

**Review by rymaju — APPROVED** [review]
Barely any code for the real backend logic is touched here. Looks 100% fine to just merge into master

File: scripts/deploy.sh [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+    exit 1
+fi
+
+# Check for updated database password
+if [[ -z "${CFC_DB_PASSWORD}" ]]; then
+    echo "ERROR: environment variable not set for database password"
+    exit 1
+fi
+
+# Check for existing db.properties file
```
Comment (liammoyn) [inline scripts/deploy.sh:19]: Shouldn't this check happen first? I assume this file is where you're getting the database username and password from

**Review by jackblanc — APPROVED** [review]
blind LGTM, review requirement has changed back to 1 but hasn't updated

**Review by liammoyn — APPROVED** [review]
Cool, can you do this for the backend scaffold as well when you get the chance?

File: scripts/deploy.sh (hunk 1 above)
Comment (justinkonecny) [reply scripts/deploy.sh:19]: This check happens before any actual logic gets executed. I think the order in which these checks happen is irrelevant, as all of them are required to make any changes to `db.properties`.

Comment (justinkonecny) [reply scripts/deploy.sh:19]: And yah, I'll make the same changes to backend scaffold.

**Pull request merged**